week_2/bus_stops_3/bus_stops_3.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <map>
+#include <set>
+#include <string>
+
+int main() {
+  int Q;
+  std::cin >> Q;
+  std::string stop;
+  int stops_count;
+  int k = 1;
+  std::set<std::string> stops;
+  std::map<std::set<std::string>, int> route;
+
+  for (int it = 0; it < Q; ++it) {
+    std::cin >> stops_count;
+
+    for (int i = 0; i < stops_count; ++i) {
+      std::cin >> stop;
+      stops.insert(stop);
+    }
+
+    if (route.count(stops) != 0) {
```

Comment:
@Ypostolov please, give example of `route`  and `stops` when `route.count(stops)` will be equal to 3

Reply:
in: stop1 stop2 stop3
out: New bus 1 
route will be '1'